src/sptezv.f

```diff
@@ -1,43 +1,38 @@
+C> @brief Perform a simple vector spherical transform
 C> @author IREDELL @date 96-02-29
 
 C> This subprogram performs a spherical transform
 C> between spectral coefficients of divergence and curl
 C> and a vector field on a global cylindrical grid.
 C> The wave-space can be either triangular or rhomboidal.
 C> The grid-space can be either an equally-spaced grid
-C> (with or without pole points) or a gaussian grid.
-C> The wave field is in sequential 'IBM ORDER'.
-C> The grid fiels is indexed east to west, then north to south.
+C> (with or without pole points) or a Gaussian grid.
+C> The wave field is in sequential 'ibm order'.
+C> The grid field is indexed east to west, then north to south.
 C> For more flexibility and efficiency, call SPTRAN().
 C> Subprogram can be called from a multiprocessing environment.
 C>
-C> @param IROMB    - INTEGER SPECTRAL DOMAIN SHAPE
-C>                (0 FOR TRIANGULAR, 1 FOR RHOMBOIDAL)
-C> @param MAXWV    - INTEGER SPECTRAL TRUNCATION
-C> @param IDRT     - INTEGER GRID IDENTIFIER
-C>                (IDRT=4 FOR GAUSSIAN GRID,
-C>                 IDRT=0 FOR EQUALLY-SPACED GRID INCLUDING POLES,
-C>                 IDRT=256 FOR EQUALLY-SPACED GRID EXCLUDING POLES)
-C> @param IMAX     - INTEGER EVEN NUMBER OF LONGITUDES.
-C> @param JMAX     - INTEGER NUMBER OF LATITUDES.
-C> @param[out] WAVED    - REAL (2*MX) WAVE DIVERGENCE FIELD IF IDIR>0
-C>                WHERE MX=(MAXWV+1)*((IROMB+1)*MAXWV+2)/2
-C> @param[out] WAVEZ    - REAL (2*MX) WAVE VORTICITY FIELD IF IDIR>0
-C>                WHERE MX=(MAXWV+1)*((IROMB+1)*MAXWV+2)/2
-C> @param[out] GRIDU    - REAL (IMAX,JMAX) GRID U-WIND (E->W,N->S) IF IDIR<0
-C> @param[out] GRIDV    - REAL (IMAX,JMAX) GRID V-WIND (E->W,N->S) IF IDIR<0
-C> @param IDIR     - INTEGER TRANSFORM FLAG
-C>                (IDIR>0 FOR WAVE TO GRID, IDIR<0 FOR GRID TO WAVE)
-C>
-C> SUBPROGRAMS CALLED:
-C>   - SPTRANFV()     PERFORM A VECTOR SPHERICAL TRANSFORM
-C>   - NCPUS()        GETS ENVIRONMENT NUMBER OF CPUS
+C> @param IROMB Spectral domain shape
+C> (0 for triangular, 1 for rhomboidal)
+C> @param MAXWV Spectral truncation
+C> @param IDRT Grid identifier
+C> - IDRT=4 for Gaussian grid
+C> - IDRT=0 for equally-spaced grid including poles
+C> - IDRT=256 for equally-spaced grid excluding poles
+C> @param IMAX Even number of longitudes
+C> @param JMAX Number of latitudes
+C> @param[out] WAVED Wave divergence field if IDIR>0
+C> where MX=(MAXWV+1)*((IROMB+1)*MAXWV+2)/2
+C> @param[out] WAVEZ Wave vorticity field if IDIR>0
+C> where MX=(MAXWV+1)*((IROMB+1)*MAXWV+2)/2
+C> @param[out] GRIDU Grid u-wind (E->W,N->S) if IDIR<0
+C> @param[out] GRIDV Grid v-wind (E->W,N->S) if IDIR<0
+C> @param IDIR Transform flag
+C> (IDIR>0 for wave to grid, IDIR<0 for grid to wave)
 C>
-C> @note MINIMUM GRID DIMENSIONS FOR UNALIASED TRANSFORMS TO SPECTRAL:
-C>   DIMENSION                    |LINEAR              |QUADRATIC
+C> @note Minimum grid dimensions for unaliased transforms to spectral:
```

Comment:
@edwardhartnett Should these tables in the subroutine doco be using "@note"? Seems like it could be "###" or just nothing.

Reply:
I usually take out the note tags for the table and some other places, and leave them in when it seems worthwhile.